kinder/pkg/cluster/manager/actions/cluster-info.go

```diff
@@ -52,12 +53,27 @@ func CluterInfo(c *status.Cluster) error {
 		// NB. before v1.13 local etcd is listening on localhost only; after v1.13
 		// local etcd is listening on localhost and on the advertise address; we are
 		// using localhost to accommodate both the use cases
-		if err := cp1.Command(
-			"kubectl", "--kubeconfig=/etc/kubernetes/admin.conf", "exec", "-n=kube-system", fmt.Sprintf("etcd-%s", c.BootstrapControlPlane().Name()),
+
+		etcdArgs := []string{
+			"--kubeconfig=/etc/kubernetes/admin.conf", "exec", "-n=kube-system", fmt.Sprintf("etcd-%s", c.BootstrapControlPlane().Name()),
 			"--",
 			"etcdctl", fmt.Sprintf("--endpoints=https://127.0.0.1:2379"),
-			"--ca-file=/etc/kubernetes/pki/etcd/ca.crt", "--cert-file=/etc/kubernetes/pki/etcd/peer.crt", "--key-file=/etc/kubernetes/pki/etcd/peer.key",
```

Comment:
this is quite the breaking change in `etcdctl`, but we should stop executing etcdctl from the etcd container, in general.
etcdctl will soon be removed from from the etcd image (due to distroless).

so the member list has to be obtained in another way.

Reply:
it's unclear, but it's already WIP.
might happen in 2020 or later.